tests/virt/node/migration_and_maintenance/test_vm_disk_load_with_migration.py

```
@@ -92,7 +92,7 @@ def get_disk_usage(ssh_exec):
 )
 @pytest.mark.s390x
 @pytest.mark.rwx_default_storage
-def test_fedora_vm_load_migration(vm_with_fio, running_fio_in_vm):
+def test_fedora_vm_load_migration(admin_client, vm_with_fio, running_fio_in_vm):
```

Ruff (0.15.14)

[warning] Unused function argument: `running_fio_in_vm`

(ARG001)